### PR TITLE
Update IAM FAQ

### PR DESCRIPTION
**Pull request by viorelaioia**

### CODE REVIEW

**Comment by viorelaioia** [comment]
@hmitsch , @gdestuynder , can you take a look? Thanks!

**Comment by hmitsch** [comment]
This looks great, thank you!

File: IAM/Frequently_asked_questions.mediawiki [hunk 1]
```diff
@@ -31,9 +31,10 @@ Email login ("passwordless") is our current solution for this use case. Some app
 ==== '''Q''': ''I would like access to specific groups, such as the NDA group, but it requires me to use a different login method, why?'' ====
 
 We only allow login, or authentication methods that can verifiably require two-factor authentication (2FA) in order to join any group that may grant you access to data that is not public, such as what we call [https://wiki.mozilla.org/Security/Data_Classification STAFF CONFIDENTIAL data].
-At the time of writing, only LDAP, Google accounts that use our LDAP backend (i.e. '''not''' '@gmail.com' accounts) and GitHub account support this functionality.
+At the time of writing, only LDAP, Google accounts that use our LDAP backend (i.e. '''not''' '@gmail.com' accounts), Firefox accounts and GitHub accounts support this functionality.
 
-Example: you could get a GitHub account with two-factor authentication enabled. Here's some documentation on how to do this: https://help.github.com/articles/about-two-factor-authentication/
+Example: you could get a GitHub account with two-factor authentication enabled. Here's some documentation on how to do this: https://help.github.com/articles/about-two-factor-authentication/. <br>
```
Comment (gdestuynder) [inline IAM/Frequently_asked_questions.mediawiki]: nit" new line is sufficient
```suggestion
Example: you could get a GitHub account with two-factor authentication enabled. Here's some documentation on how to do this: https://help.github.com/articles/about-two-factor-authentication/.
```

File: IAM/Frequently_asked_questions.mediawiki [hunk 2]
```diff
@@ -56,14 +57,14 @@ Thanks for your help!
...
-==== '''Q''': ''How can I set up two-factor authentication (2FA) for my github account?'' ====
+==== '''Q''': ''How can I set up two-factor authentication (2FA) for my github account, using an app on my phone (Android/iOS/Blackberry)?'' ====
 
 Two-factor authentication can be configured by using a Time-based One-Time Password (TOTP) application, which automatically generates an authentication code that changes after a certain period of time. See configuration steps below.<br>
 1.  Download one of these apps:
 *For Android, iOS, and Blackberry: Google Authenticator
 *For Android and iOS: Duo Mobile
 *For Windows Phone: Authenticator
-2.  In the upper-right corner of any page, click your profile photo, then click Settings.
+2.  In the upper-right corner of any github page, click your profile photo, then click Settings.
```
Comment (gdestuynder) [inline IAM/Frequently_asked_questions.mediawiki]:
```suggestion
2.  In the upper-right corner of any GitHub page, click your profile photo, then click Settings.
```

File: IAM/Frequently_asked_questions.mediawiki [hunk 3]
```diff
@@ -102,41 +103,49 @@ Two-factor authentication can be configured by using a Time-based One-Time Passw
...
    [[File:Mozillians_-_login_with_github_to_upgrade_account.png|350px]]
 5.  Enter 2fa code from your application.
    [[File:Mozillians_-_enter_2fa_code.png|300px]]
 6.  Navigate to Settings -> Profile Identities section, and verify that Github is set as your login identity. That means this is the only account you can use from now on to login to mozillians.
    [[File:Mozillians_-_upgrade_to_github.png|350px]]
 7.  Trying to login with email to mozillians will return an error page, asking to login with github.
-   [[File:Mozillians_-_forbidden_page_message_-_github.png|350px]]
+   [[File:2-_error.png|350px]]
+
+==== '''Q''': ''The email address I use to login to my mozillians account matches the primary email of my Firefox Accounts account. How can I upgrade my mozillians account from passwordless to Firefox Accounts?'' ====
```
Comment (gdestuynder) [inline IAM/Frequently_asked_questions.mediawiki]:
```suggestion
==== '''Q''': ''The email address I use to login to my Mozillians account matches the primary email of my Firefox Accounts account. How can I upgrade my Mozillians account from Passwordless to Firefox Accounts?'' ====
```

File: IAM/Frequently_asked_questions.mediawiki [hunk 4]
```diff
@@ -102,41 +103,49 @@ Two-factor authentication can be configured by using a Time-based One-Time Passw
...
+
+1.  In the following steps we assume you have 2FA set for your Firefox Accounts account. If not, see the steps from [https://blog.mozilla.org/services/2018/05/22/two-step-authentication-in-firefox-accounts/ here].<br>
```
Comment (gdestuynder) [inline IAM/Frequently_asked_questions.mediawiki]:
```suggestion
1.  In the following steps we assume you have 2FA set for your Firefox Accounts account. If not, see the steps from [https://blog.mozilla.org/services/2018/05/22/two-step-authentication-in-firefox-accounts/ here].
```

File: IAM/Frequently_asked_questions.mediawiki [hunk 5]
```diff
@@ -102,41 +103,49 @@ Two-factor authentication can be configured by using a Time-based One-Time Passw
...
+2.  Navigate to mozillians page and click Log In/Sign Up button. <br>
```
Comment (gdestuynder) [inline IAM/Frequently_asked_questions.mediawiki]:
```suggestion
2.  Navigate to mozillians page and click Log In/Sign Up button. 
```

File: IAM/Frequently_asked_questions.mediawiki [hunk 6]
```diff
@@ -102,41 +103,49 @@ Two-factor authentication can be configured by using a Time-based One-Time Passw
...
+3.  Select “Continue with Firefox” method from mozillians login page. <br>
```
Comment (gdestuynder) [inline IAM/Frequently_asked_questions.mediawiki]:
```suggestion
3.  Select “Continue with Firefox” method from mozillians login page.
```

File: IAM/Frequently_asked_questions.mediawiki [hunk 7]
```diff
@@ -102,41 +103,49 @@ Two-factor authentication can be configured by using a Time-based One-Time Passw
...
+   [[File:3_-_moz_login.png|350px]]
+4.  Enter Firefox Accounts credentials.
+   [[File:5_-_fxa_login.png|350px]]
+5.  Enter 2fa code from your application.
+   [[File:9_-_fxa_2fa.png|300px]]
+6.  Navigate to Settings -> Profile Identities section, and verify that Firefox Accounts is set as your login identity. That means this is the only account you can use from now on to login to mozillians.
+   [[File:8_-_fxa_primary.png|350px]]
+7.  Trying to login with email to mozillians will return an error page, asking to login with Firefox Accounts.
```
Comment (gdestuynder) [inline IAM/Frequently_asked_questions.mediawiki]:
```suggestion
7.  Trying to login with email to Mozillians will return an error page, asking to login with Firefox Accounts.
```

File: IAM/Frequently_asked_questions.mediawiki [hunk 8]
```diff
@@ -186,15 +197,35 @@ If you own a volunteer LDAP account, then you can upgrade your mozillians profil
...
-9.  Scroll down to “Profile Identities” section and verify that your Google account is set as your login identity.
+9.  Scroll down to “Profile Identities” section and verify that your Google account is in your Contact identities section.
    [[File:Mozillians_-_google_identity.png|350px]]
+
+
+==== '''Q''': ''What issues I might encounter by upgrading to Firefox Accounts?'' ====
+
+There are some known issues with using Firefox Accounts in Mozilla IAM: 
+
+1.  Login with Firefox Accounts is unavailable from inside some Android applications, including IRCCloud and Slack. This is due to lack of localStorage support in some Android WebViews. <br>
```
Comment (gdestuynder) [inline IAM/Frequently_asked_questions.mediawiki]:
```suggestion
1.  Login with Firefox Accounts is unavailable from inside some Android applications, including IRCCloud and Slack. This is due to lack of localStorage support in some Android WebViews.
```

Comment (gdestuynder) [reply IAM/Frequently_asked_questions.mediawiki]: do we have a link to their issue?

**Comment by gene1wood** [comment]
I've deployed this by hand at https://wiki.mozilla.org/index.php?title=IAM/Frequently_asked_questions&oldid=1206179

**Pull request merged**